Makefile

```diff
@@ -9,7 +9,7 @@ IMAGETAG ?= latest
 CABUNDLECONFIGMAP ?= webhook-cert
 VWC_ANNOTATION ?= managed.openshift.io/inject-cabundle-from
 
-IMG ?= quay.io/lseelye/python3-webhookbase
+IMG ?= quay.io/cshereme/python3-webhookbase
```

Comment:
@lisa where should we be pushing these images?

Reply:
@csheremeta I believe the changes made by @robotmaxtron to the build scripts (eg in build/*.sh) handle where the images end up. In many ways the Makefile definition is obsoleted now.